test/show.jl

```diff
@@ -65,151 +64,115 @@ end
     show(io, df_big)
     str = String(take!(io.io))
     @test str == """
-    25×5 DataFrame. Omitted printing of 2 columns
-    │ Row │ x1       │ x2       │ x3       │
-    │     │ Int64    │ Int64    │ Int64    │
-    ├─────┼──────────┼──────────┼──────────┤
-    │ 1   │ 10000001 │ 10000026 │ 10000051 │
-    ⋮
-    │ 24  │ 10000024 │ 10000049 │ 10000074 │
-    │ 25  │ 10000025 │ 10000050 │ 10000075 │"""
+        25×5 DataFrame
+         Row │ x1        x2        x3        x ⋯
+             │ Int64     Int64     Int64     I ⋯
+        ─────┼──────────────────────────────── ⋯
+         1   │ 10000001  10000026  10000051  1 ⋯
+         2   │ 10000002  10000027  10000052  1 ⋯
+         3   │ 10000003  10000028  10000053  1 ⋯
+         ⋮   │ ⋮         ⋮         ⋮         ⋮ ⋯"""
```

Comment:
This is what I call @nalimilan's drill (which is always welcome 😄).

Reply:
Hum, good catch. The elipses in horizontal bar is not good at all. I will try to remove that from PrettyTables. With respect to the number of ellipsis, I really like to add it to all columns. Currently, there is no way to change this in PrettyTables, but should be very easy if you want something different. Hence, what should I do?

Reply:
Can we negotiate an intermediate position? :-p Keep ellipses for all columns, but only for some of the rows.

Reply:
Sure! I need to add this feature to PrettyTables. Currently, there is a printing function that, if it reaches the end of screen, then it add the ellipses. I need to modify so that we can select a parameter to omit the ellipses depending on the line count. It does not seem difficult to add.

Reply:
Can I always add ellipses to the header and sub-header but add an option to skip `n` lines in the table data?

Reply:
Sure.

Reply:
Here is what I have with a new feature I just introduced in PrettyTables. In this case, I am skipping 3 lines when printing ellipsis.

```julia
julia> df_big = DataFrame(reshape(Int64(10000001):Int64(10000000+25*5), 25, 5));

julia> io = IOContext(IOBuffer(), :displaysize=>(15,40), :limit=>true);

julia> show(io, df_big);

julia> str = String(take!(io.io));

julia> print(str)
25×5 DataFrame
 Row │ x1        x2        x3        x ⋯
     │ Int64     Int64     Int64     I ⋯
─────┼────────────────────────────────
 1   │ 10000001  10000026  10000051  1 ⋯
 2   │ 10000002  10000027  10000052  1
 3   │ 10000003  10000028  10000053  1
 4   │ 10000004  10000029  10000054  1
 5   │ 10000005  10000030  10000055  1 ⋯
 6   │ 10000006  10000031  10000056  1
 7   │ 10000007  10000032  10000057  1
 ⋮   │ ⋮         ⋮         ⋮         ⋮ ⋱
```

I just need to check some particular cases.

Reply:
Looks cool!

Just a small nitpick: the vertical bar continues on the row where ellipses are printed, but the horizontal bar stops before the ellipses. It would make sense to be consistent (not sure which is better).